tests/flytekit/unit/models/test_dynamic_wfs.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+from flytekit.sdk import tasks as _tasks, workflow as _workflow
+from flytekit.sdk.types import Types as _Types
+from flytekit.common import constants as _sdk_constants
+
+
+@_tasks.inputs(num=_Types.Integer)
+@_tasks.outputs(out=_Types.Integer)
+@_tasks.python_task
+def inner_task(wf_params, num, out):
+    wf_params.logging.info("Running inner task... setting output to input")
+    out.set(num)
+
+
+@_workflow.workflow_class()
+class IdentityWorkflow(object):
```

Comment:
can you add a test in `user_scenarios/unit_testing` where a dynamic task blows up with a useful exception if you try to execute a unit_test on a dynamic task with a launch plan or sub-workflow

Reply:
I have a test, and it doesn't blow up, because I added that bit that was obviously very confusing about how it only runs half the steps... if a launch plan or workflow node is detected, the unit test will end early.  I like being able to test the dynamic job spec that's generated, but I also don't want to/don't know how to finish the entire dynamic task unit test engine right now that can follow through on lps/subwfs.

Reply:
ah ok got it.